### PR TITLE
Feat: Add Comment Out to Issue Templates

### LINKED ISSUE #4: feat: Comment Out Instructions in the Issue Templates
# Suggestion/Improvement

🌟 Hi there! We're excited to hear your suggestions and ideas for making this project better. Thank you for contributing to our community!

## Description

After previewing a written issue, commenting out the questions/instuctions in the issue templates would give more clarity to read the issue.

## Motivation

When a contributor doesn't delete the instructions when they write an issue, it makes harder for maintainers/triage team to review the issue. But commenting out the issue would not give any problems for contributors.

### Screenshots

Below are examples screenshot of result before and after commenting out the instructions:

**Before commenting out** (read arrow pointing at the issue that contributors have written):

without comment

**After commenting out**:

comment out

## Implementation (if applicable)

Comment out the instructions like example below:

```markdown
<!-- Comment this out -->
``` 

## Your Participation Matters

We appreciate your efforts to make this project more awesome! Your suggestions are valuable to us, and we'll review them as soon as possible.

### PR DESCRIPTION
**Pull request by adiati98**
## Description

This PR adds comment out to instructions in issue templates for better readable result.

## Related Issue

Closes #4

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -20,19 +19,18 @@ Describe the bug in a few sentences. What happened, and what were you expecting?
...
 ## Anything Else?
 
-Feel free to share any additional details or thoughts.
+<!-- Feel free to share any additional details or thoughts. -->
 
 Thank you for your contribution! We'll look into this as soon as possible.
 
 ---
 
 **Note:** Please follow our [Code of Conduct](https://shesharp.co/code-of-conduct) during your participation in this community.
```
Comment (saragerion) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]: Can you make also this section a comment?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -10,18 +9,16 @@ about: Share your ideas to help us improve
...
+<!-- Help us understand why you think this suggestion is valuable. How would it benefit the project or community? -->
 
 ## Implementation (if applicable)
 
-If you have specific ideas on how this suggestion could be implemented, share them here. Code snippets, screenshots, or any additional context are welcome!
+<!-- If you have specific ideas on how this suggestion could be implemented, share them here. Code snippets, screenshots, or any additional context are welcome! -->
 
 ## Your Participation Matters
 
 We appreciate your efforts to make this project more awesome! Your suggestions are valuable to us, and we'll review them as soon as possible.
```
Comment (saragerion) [inline .github/ISSUE_TEMPLATE/feature_request.md:25]: Can you make also this section a comment?

**Comment by saragerion** [comment]
Thanks for the PR! left a few comments (as minor suggestions)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (adiati98) [reply .github/ISSUE_TEMPLATE/bug_report.md:37]: To confirm, all the line 32-36, right, @saragerion?

Comment (saragerion) [reply .github/ISSUE_TEMPLATE/bug_report.md:37]: Correct :)

**Comment by adiati98** [comment]
@saragerion it's done 👍

**Comment by adiati98** [comment]
Thank you, @saragerion! 💙

**Pull request merged**